test/DynamoCoreTests/TestZeroTouchClass.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Dynamo.DSEngine;
+using Dynamo.Library;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using Dynamo.Search.SearchElements;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
+
+namespace ProtoTest.FFITests
+{
+    [TestFixture]
+    class TestZeroTouchClass
+    {
+        [Test]
+        public void DescriptionTest()
+        {
+            var assembly = System.Reflection.Assembly.UnsafeLoadFrom("FFITarget.dll");
+            var testClass = assembly.GetType("FFITarget.DummyZeroTouchClass");
+
+            MethodInfo methodWithDesc = testClass.GetMethod("FunctionWithDescription");
+            MethodInfo methodWithoutDesc = testClass.GetMethod("FunctionWithoutDescription");
+
+            NodeDescriptionAttribute atr = new NodeDescriptionAttribute("");
+            IEnumerable<TypedParameter> arguments;
+            FunctionDescriptor fucDescriptor;
+
+            // 1 case. Method with description.
+            var attributes = methodWithDesc.GetCustomAttributes(typeof(NodeDescriptionAttribute), false);
+            if(attributes.Length >0)
+                atr = attributes[0] as NodeDescriptionAttribute;
```

Comment:
Hi @aosyatnik, since we are creating this test case based on a test class, it must contain the `NodeDescriptionAttribute`, because we *need* our test data to be matching our expectation exactly so for this to be a meaningful test. So here we should do:

```csharp
Assert.IsNotNull(attributes);
Assert.IsGreaterThan(0, attributes.Length);
var atr = attributes[0] as NodeDescriptionAttribute;
```